pages/apim/3.x/how-tos/how-to-enable-opentracing-with-jaeger.adoc

```diff
@@ -1,102 +1,104 @@
-
-Here, you can change the variables as needed. 
-
-image:apim/3.x/how-tos/enable-opentracing/environment-variables-closeup.png[Environment variables in Jaeger Gateway, fig. 2.] 
-
-
-== How To Enable OpenTracing Via Docker Compose 
-
-Enabling OpenTracing is easy -- get started using Jaegar as a tracer with this Docker Compose. 
+Since the Jaeger tracer is not bundled with APIM by default,
+link:https://download.gravitee.io/#graviteeio-apim/plugins/tracers/gravitee-tracer-jaeger/[click here^] to download the Gravitee Jaeger Tracer .ZIP file, which adds the tracer to the configuration of your APIM Gateway instance.
+// more info needed
 
-With this option enabled, you can continue to call your APIs through your gateway with the usual host: `http://localhost:8082/myapi`.
+=== 5. Test OpenTracing
 
-=== 1. How To Run OpenTracing With Jaeger
+. Now that you have OpenTracing enabled, run your API Gateway and APIM.
+. Open APIM and choose an API that already has a policy assigned to it (or create a test API and add any policies you like for this test). Now, call your API.
+. To see your calls:
+    * Open the JaegerUI by visiting http://localhost:16686.
 
-* 1. Since the Jaeger tracer is not bundled by default, **you must download the .ZIP file** for the version you want to run.
+    NOTE: JaegerUI was automatically installed when you ran the Docker image.
 
-* 2. Download the .ZIP here: https://download.gravitee.io/#graviteeio-apim/plugins/tracers/gravitee-tracer-jaeger/. 
+    * Click *Search* and find the API you called.
+    +
+    image:apim/3.x/how-tos/enable-opentracing/tracing-info-general.png[General tracing info from JaegerUI].
 
-* 3. After downloading, **you must copy this into the `opentracing-jaeger/.plugins` directory using the command below:** 
+    In these examples, you’ll also notice that each policy traversed is listed on a different line to provide greater visibility and assist with debugging. You can even see timestamp info on the detailed view.
+    +
+    image:apim/3.x/how-tos/enable-opentracing/tracing-info-detailed.png[Detailed tracing info from JaegerUI].
 
-`APIM_VERSION={APIM_VERSION} docker-compose up -d` 
 
-* 4. Be sure to fetch last version of images by running this command: 
-`export APIM_VERSION={APIM_VERSION} && docker-compose down -v && docker-compose pull && docker-compose up`
+== Configure Environment Variables
 
-== How Can I See My Traces? 
+You can configure Jaeger OpenTracing with environment variables in the Jaeger Gateway. Instead of configuring `gravitee.yml` directly, you can use its values as a basis for adding environment variables to the Jaeger Gateway:
 
-Jaeger comes with a helpful, user-friendly UI that allows you to see your calls. To access this UI, visit http://localhost:16686.
+image:apim/3.x/how-tos/enable-opentracing/environment-variables-large.png[Environment variables in Jaeger Gateway.]
```

Comment:
we should not have a screenshot here (not readable and you can see all my local config which is useless
```suggestion
`gravitee_services_tracing_enabled=true`
`gravitee_services_tracing_jaeger_host=localhost`
`gravitee_services_tracing_jaeger_port=14250`
```